docs/zh/operations/table_engines/collapsingmergetree.md

```diff
@@ -21,22 +21,22 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 [SETTINGS name=value, ...]
 ```
 
-For a description of request parameters, see [request description](../../query_language/create.md).
```

Comment:
It is different from https://github.com/yandex/ClickHouse/pull/5168/files#diff-e98acaf76ea56d6ac0c606763b49263cR24. maybe there is no synchronization of en documents before translation?

Reply:
I‘m sorry for the negligence, there were five differences but i just found four and synchronized then before. I 'll be more careful next time.
Really thanks for your review.

Reply:
It seems that this line does't have any difference。 Or is there anything i haven't noticed?